### PR TITLE
Update to use 2.4 preview package.

### PR DESCRIPTION
**Pull request by ranjeshj**
Updating to use the latest 2.4 pre-release package (2.4.0-prerelease.191217001)

### CODE REVIEW

File: XamlControlsGallery/XamlControlsGallery.csproj [hunk 1]
```diff
@@ -1,1148 +1,1148 @@
...
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-200.png" />
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-400.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-100.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-125.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-150.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-200.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-400.png" />
+    <Content Include="Assets\Tiles\WideTile.scale-100.png" />
+    <Content Include="Assets\Tiles\WideTile.scale-125.png" />
+    <Content Include="Assets\
```
Comment (stmoy) [inline XamlControlsGallery/XamlControlsGallery.csproj]: Why isn't this the only line that has changed?

Comment (ranjeshj) [reply XamlControlsGallery/XamlControlsGallery.csproj]: Weird. Visual studio and codeflow show only one line change.

Comment (ranjeshj) [reply XamlControlsGallery/XamlControlsGallery.csproj]: I fixed it.

**Comment by chingucoding** [comment]
I think this can be closed since #318 updates to a newer version of WinUI than this PR would.

**Pull request closed without merging**